include/entity.h

```diff
@@ -519,6 +519,7 @@ typedef struct {
     /* 0xA6 */ s16 unkA6;
 } ET_GurkhaSword;
 
+struct draculaPrimitive;
```

Comment:
Is this here by accident?

Reply:
I noticed this warning when compiling and get it in my generated context as well. I can move it into a separate PR.